src/transformers/commands/info.py

```diff
@@ -0,0 +1,61 @@
+import platform
+from argparse import ArgumentParser
+
+from transformers import __version__ as version
+from transformers.commands import BaseTransformersCLICommand
+
+
+def info_command_factory(_):
+    return InfoCommand()
+
+
+class InfoCommand(BaseTransformersCLICommand):
+    @staticmethod
+    def register_subcommand(parser: ArgumentParser):
+        download_parser = parser.add_parser("info")
+        download_parser.set_defaults(func=info_command_factory)
+
+    def run(self):
+        pt_version = "not installed"
+        pt_cuda_available = "NA"
+        try:
+            import torch
+
+            pt_version = torch.__version__
+            pt_cuda_available = torch.cuda.is_available()
+        except ImportError:
+            pass
+
+        tf_version = "not installed"
+        tf_cuda_available = "NA"
+        try:
+            import tensorflow as tf
+
+            tf_version = tf.__version__
+            try:
+                # deprecated in v2.1
+                tf_cuda_available = tf.test.is_gpu_available()
+            except AttributeError:
+                # returns list of devices, convert to bool
+                tf_cuda_available = bool(tf.config.list_physical_devices("GPU"))
+        except ImportError:
+            pass
```

Comment:
_Suggestion_ -> rely on transformers methods to check platform availability:

```python
from transformers import is_torch_available, is_tf_available

if is_torch_available():
...

if is_tf_available():
...
```

Reply:
Thanks, that make sense. Changed in https://github.com/huggingface/transformers/pull/2699/commits/f0d5aded94b2e6e80ca31ace8fc6ce82aba75ab9